pkg/graphviz/traceflow_test.go

```diff
@@ -315,6 +663,21 @@ Delivered", shape=box, style="rounded,filled,solid" ];
 			traceflow:      tfInOneNode,
 			expectedOutput: expectedOutputInOneNode,
 		},
+		{
+			name:           "live traceflow with source IP two nodes",
```

Comment:
Ditto, please check other occurrences.
```suggestion
			name:           "live traceflow with source IP two Nodes",
```